test/hotspot/jtreg/compiler/intrinsics/math/TestCopySignIntrinsic.java

```diff
@@ -73,28 +76,14 @@ public TestCopySignIntrinsic() {
         adresult = new double[SIZE];
         edresult = new double[SIZE];
 
-        IntStream.range(0, SIZE - 8).forEach(i -> { fmagnitude[i] = rd.nextFloat(-Float.MAX_VALUE, Float.MAX_VALUE); });
-        IntStream.range(0, SIZE - 8).forEach(i -> { dmagnitude[i] = rd.nextFloat(-Float.MAX_VALUE, Float.MAX_VALUE); });
-        IntStream.range(0, SIZE).forEach(i -> { fsign[i] = rd.nextFloat(-Float.MAX_VALUE, Float.MAX_VALUE); });
-        IntStream.range(0, SIZE).forEach(i -> { dsign[i] = rd.nextFloat(-Float.MAX_VALUE, Float.MAX_VALUE); });
-
-        fmagnitude[SIZE - 1] = Float.NaN;
-        fmagnitude[SIZE - 2] = Float.NaN;
-        fmagnitude[SIZE - 3] = 0.0f;
-        fmagnitude[SIZE - 4] = -0.0f;
-        fmagnitude[SIZE - 5] = Float.NEGATIVE_INFINITY;
-        fmagnitude[SIZE - 6] = Float.POSITIVE_INFINITY;
-        fmagnitude[SIZE - 7] = -Float.MIN_VALUE;
-        fmagnitude[SIZE - 8] = Float.MIN_VALUE;
-
-        dmagnitude[SIZE - 1] = Double.NaN;
-        dmagnitude[SIZE - 2] = Double.NaN;
-        dmagnitude[SIZE - 3] = 0.0;
-        dmagnitude[SIZE - 4] = -0.0;
-        dmagnitude[SIZE - 5] = Double.NEGATIVE_INFINITY;
-        dmagnitude[SIZE - 6] = Double.POSITIVE_INFINITY;
-        dmagnitude[SIZE - 7] = -Double.MIN_VALUE;
-        dmagnitude[SIZE - 8] = Double.MIN_VALUE;
+        Generator<Float> genFloat = G.floats();
+        Generator<Double> genDouble = G.doubles();
+        for (int i = 0; i < SIZE; i++) {
+            fmagnitude[i] = genFloat.next();
+            dmagnitude[i] = genFloat.next();
+            fsign[i]      = genFloat.next();
+            dsign[i]      = genFloat.next();
+        }
```

Comment:
Two comments:
- You probably wanted to use the double generator for the double arrays, right?
- You can fill a whole array directly with e.g. `Generators.G.fill(genFloat, fmagniture)`.